assets/js/scenes.js

```diff
@@ -43,6 +44,33 @@ const generateScenes = () => {
 
     // spawn baby yoda
     spawnBabyYoda();
+    
+    // display score
+    add([
+        text("Score:0"),
+        pos(10, 0),
+        layer("ui"),
+        {
+          value: score,
+        },
+        scale(0.4),
+    ]);
+    // Display time left
+    add([
+      text("Time left:" + parseInt(tl)),
+      pos(550, 0),
+      layer("ui"),
+      scale(0.4),
+    ]);
+
+    // Display eggs collected
+    add([
+      text("Eggs collected:" + parseInt(eggs)),
```

Comment:
Suggestion: You might also be able to make use of [JavaScript template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here. They allow you to stay in the string and add the variables directly. The way you do that is by opening the string with backticks (as opposed to quotation marks) and then adding your JavaScript wrapped like this: `${javascript goes here}`.

Reply:
Thanks Stephen this is useful too - I didn't think of it in a JS context... more of bash context but makes perfect sense - Many thanks